addon/serializers/application.js

```diff
@@ -190,8 +190,8 @@ export default DS.RESTSerializer.extend({
       hasMany.forEach( function( child ) {
         json[key].objects.push({
           '__type'    : 'Pointer',
-          'className' : child.parseClassName(),
-          'objectId'  : child.get( 'id' )
+          'className' : _this.parseClassName(child.type.typeKey),
```

Comment:
In `serializeBelongsTo` the `key` (used in serialization to the server) is passed to `parseClassName`. Here however the `typeKey` is passed, the Ember-Data internal value.

IMO only the `typeKey` should be passed to `parseClassName`, but can you clarify?